CatAnalyzer/test/run_TtbarBbbarDiLeptonAnalyzer_cfg.py

```diff
@@ -0,0 +1,134 @@
+    combineBy = cms.string("or"),
+    triggersToMatch = cms.vstring(
+        "HLT_Mu17_TrkIsoVVL_Ele12_CaloIdL_TrackIdL_IsoVL_v",
+        "HLT_Mu8_TrkIsoVVL_Ele17_CaloIdL_TrackIdL_IsoVL_v",
+    ),
+    doFilter = cms.bool(False),
+)
+
+process.filterTrigELEL = process.filterTrigMUEL.clone(
+    triggersToMatch = cms.vstring(
+        "HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_v",
+        "HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v",
+    ),
+)
+
+process.filterTrigMUMU = process.filterTrigMUEL.clone(
+    triggersToMatch = cms.vstring(
+#      "HLT_Mu17_Mu8_DZ_v",
+#      "HLT_Mu17_TkMu8_DZ_v",
+      "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v",
+      "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_v",
+      "HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_v",
+    ),
+)
+
+#process.load("CATTools.CatAnalyzer.ttbarDileptonKinSolutionAlgos_cff")
+
+process.cattree = cms.EDAnalyzer("TtbarBbbarDiLeptonAnalyzer",
+    recoFilters = cms.InputTag("filterRECO"),
+    nGoodVertex = cms.InputTag("catVertex","nGoodPV"),
+    genweight = cms.InputTag("genWeight","genWeight"),
+    genweightQ = cms.InputTag("genWeight","Q"),
+    genweightPDF = cms.InputTag("genWeight","pdfWeights"),
+
+    puweight = cms.InputTag("pileupWeight"),
+    puweightUp = cms.InputTag("pileupWeight","up"),
+    puweightDown = cms.InputTag("pileupWeight","dn"),
+    trigMUEL = cms.InputTag("filterTrigMUEL"),
+    trigMUMU = cms.InputTag("filterTrigMUMU"),
+    trigELEL = cms.InputTag("filterTrigELEL"),
+
+    vertices = cms.InputTag("catVertex"),
+    muons = cms.InputTag("catMuons"),
+    electrons = cms.InputTag("catElectrons"),
+    jets = cms.InputTag("catJets"),
+    mets = cms.InputTag(catmet),
+    mcLabel = cms.InputTag("prunedGenParticles"),
+    
+    partonTop_channel = cms.InputTag("partonTop","channel"),
+    partonTop_modes = cms.InputTag("partonTop", "modes"),
+    partonTop_genParticles = cms.InputTag("partonTop"),
+
+    pseudoTop = cms.InputTag("pseudoTop"),
+
+    genTtbarId = cms.InputTag("GenTtbarCategories", "genTtbarId"),
+    genTtbarId30 = cms.InputTag("GenTtbarCategories30", "genTtbarId"),
+    genTtbarId40 = cms.InputTag("GenTtbarCategories40", "genTtbarId"),
+
+    GenJets = cms.InputTag("slimmedGenJets"),
+    #solver = process.ttbarDileptonKinAlgoPSetCMSKin,
+    #solver = process.ttbarDileptonKinAlgoPSetDESYSmeared,
+    #solver = process.ttbarDileptonKinAlgoPSetDESYMassLoop,
+)
+#process.cattree.solver.tMassStep = 1
+#if cms.string('DESYSmeared') == process.cattree.solver.algo:
+#    process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
+#        cattree = cms.PSet(
+#            initialSeed = cms.untracked.uint32(123456),
+#            engineName = cms.untracked.string('TRandom3')
+#        )
+#    )
+
+process.TFileService = cms.Service("TFileService",
+    fileName = cms.string("cattree.root"
+))
+
+process.p = cms.Path(process.cattree)
+process.MessageLogger.cerr.FwkReport.reportEvery = 50000
+
+from customise_cff import *
+customise(process)
```

Comment:
?? then why do you need customise_cff? you can just put all the contents of customise_cff here.